.ci/bump-opbeans-node.groovy

```diff
@@ -10,7 +10,6 @@ pipeline {
     BASE_DIR = 'src/github.com/elastic'
     NOTIFY_TO = credentials('notify-to')
     PIPELINE_LOG_LEVEL = 'INFO'
-    PATH = "${env.PATH}:${env.WORKSPACE}/bin"
```

Comment:
REVIEW NOTE: There is no `$WORKSPACE/bin` dir, so it is misleading to add it to the PATH.